src/ReviewService/ReviewService.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<LangVersion>10.0</LangVersion>
+		<TargetFrameworks>net6.0-android;net6.0-ios;net6.0-windows10.0.19041</TargetFrameworks>
+		<!-- See https://github.com/dotnet/maui/issues/5886 for more details. -->
+		<EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<!-- TODO: Remove this when it's a NuGet. -->
```

Comment:
Should this be removed now, then? Especially since we're on a nuget repo

Reply:
Oh yes! That's right! Thanks!

Reply:
Is it still there? The TODO is still there.

Reply:
Removed in the second commit.